### ISSUE TITLE
R version for "lengths" is incorrect

### ISSUE THREAD

**Issue by berndbischl** [mllg/backports#13]
I have this

```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.2 LTS
```

lengths is already included, now backports shadows it

**Comment by berndbischl** [comment]
it was introduced in 3.2.0

ftp://cran.r-project.org/pub/R/doc/manuals/r-devel/NEWS.html

**Comment by mllg** [comment]
Thanks, fixed in 1.1.1.
